sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,408 @@
+            In pre-silicon, they can be part of the same program.
+
+            - Switch to LC TAP at runtime.
+            - Request transition to DEV or PROD state (test both cases).
+            - Perform reset by toggling POR pin.
+            - Verify isolated flash info partition expected data.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_cp_test_lock
+      desc: '''Verify provisioning and use of unlock tokens.
+
+            In transit devices will need to be switched into TEST_LOCKED states to disable debug
+            access and code execution. This test verifies that a TEST_UNLOCKED token can be
+            provisioned as part of this process.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Program TEST_UNLOCK_TOKEN in OTP transition via JTAG.
+            - Select LC TAP via strap pins (at runtime).
+            - Request transition into TEST_LOCKED state.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc state.
+
+            Implement the following steps in a way in which it could be executed at a later time
+            in silicon targets:
+
+            - Load OTP state generted by previous steps.
+            - Select LC TAP via strap pins.
+            - Request transition to TEST_UNLCOKED by providing the TEST_UNLOCK_TOKEN provisioned
+              in previous steps.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc_state.
+
+            Repeat the same flow for the number of supported TEST_LOCK/UNLOCKED states.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_exit_token
+      desc: '''Verify provisioning and use of test exit token.
+
+            The test exit token is used to move the device from test mode into DEV, PROD or
+            PROD_END state. This test also incorporates enabling of ROM execution before
+            triggering the lc_state change.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Program TEST_EXIT_TOKEN in OTP transition via JTAG.
+            - Reset device by toggling POR pin.
+            - Select CPU JTAG via strap pins.
+            - Program CREATOR_SW_CFG_ROM_EXEC_EN in OTP via JTAG.
+            - Select LC TAP via strap pins (at runtime).
+            - Request transition into {pick one: DEV, PROD, PROD_END} state providing TEST_EXIT_TOKEN.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc state.
```

Comment:
`chip_sw_lc_walkthrough` to update sequence to use TAP. May also want to check ROM execution enable with production ROM. The later could be covered by a ROM E2E test.

Reply:
Consider adding a separate test case to exercise the test_ext + bootstrap functionality.